docs/pages/web/pulsar.js

```diff
@@ -44,6 +53,128 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
         </MainSection.Subsection>
       </MainSection>
 
+      <MainSection name="Best Practices">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description={`
+            Show Pulsar with an educational [Popover](/web/popover) that contains an affordance to dismiss both Popover and Pulsar.
+            `}
+            sandpackExample={
+              <SandpackExample
+                code={doPopover}
+                name="Do - Popover"
+                hideEditor
+                previewHeight={bestPracticesHeight}
+              />
+            }
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Use Pulsar to highlight an element with no context or way to dismiss. The user should always know why something is being highlighted and be able to opt out."
+            sandpackExample={
+              <SandpackExample
+                code={dontPopover}
+                name="Don't - Popover"
+                hideEditor
+                previewHeight={bestPracticesHeight}
+              />
+            }
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use Pulsar to highlight or promote a feature or element on a surface."
+            sandpackExample={
+              <SandpackExample
+                code={doHighlight}
+                name="Do - Highlight"
+                hideEditor
+                previewHeight={bestPracticesHeight}
+              />
+            }
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description={`
+            Display Pulsar to point out an error or status update. Use [SlimBanner](/web/slimbanner) or other [messaging components](/foundations/messaging/overview) instead.
+            `}
+            sandpackExample={
+              <SandpackExample
+                code={dontStatus}
+                name="Don't - Status"
+                hideEditor
+                previewHeight={bestPracticesHeight}
+              />
+            }
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Prioritize Pulsars to call out the most critical feature users should be aware of."
+            sandpackExample={
+              <SandpackExample
+                code={doPrioritize}
```

Comment:
Should we add here a POpover to not contradict the rule of no Pulsar without context?

Reply:
it had one, just wasn't showing up 🤦